examples/guestbook/README.md

```diff
@@ -199,81 +276,51 @@ If you want to see the logs for a given container, you can run:
 $ docker logs <container_id>
 ```
 
-### Step Two: Fire up the redis master service
+### Step Three: Create the redis slave service
```

Comment:
It would be great if we could combine steps 3 and 4, since they are so similar to steps 1 and 2, and it would show off more capabilities of kubectl.

We could then put the service and RC into the same file.

We should link to: https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/managing-deployments.md